addon/components/basic-dropdown.js

```diff
@@ -163,8 +163,10 @@ export default Component.extend({
     if (!this.get('publicAPI.isOpen')) { return; }
     let onClose = this.get('onClose');
     if (onClose && onClose(this.get('publicAPI'), e) === false) { return; }
-    this.set('publicAPI.isOpen', false);
-    this.setProperties({ _verticalPositionClass: null, _horizontalPositionClass: null });
+    if (!this.isDestroying && !this.isDestroyed) {
```

Comment:
This is something that intrigues me. I don't see any possible situation where this method can be called once the component is destroyed. Did you find one?